tests/plugins/colorbutton/colorbutton.js

```diff
@@ -164,6 +164,33 @@
 
 				wait();
 			} );
-		}
+		},
+
+		// (#1084)
+		'test changing text color to automatic': testAutomaticColor(),
+
+		// (#1084)
+		'test changing background color to automatic': testAutomaticColor( true )
 	} );
+
+	function testAutomaticColor( background ) {
```

Comment:
WDYT about changing argument's name to `isBackgroundColor`?
This variable works as a flag and such name might be more suitable here.